File: swagger/apis/usertoken_api.py
from .base_api import BaseApi

from ..models.issue_request import IssueRequest
from ..models.issue_response import IssueResponse

class UsertokenApi(BaseApi):
	def __init__(self, site_id, authorization):
		super().__init__('usertoken', site_id, authorization)

	def post_issue(self, request):
		pass

	def delete_revoke(self):
		pass

File: swagger/apis/UsertokenApi.php
<?php

namespace App\Mindbody\Apis;

use App\Mindbody\Models\IssueRequest;
use App\Mindbody\Models\IssueResponse;

class UsertokenApi extends BaseApi
{
	/**
	 * Constructor
	 */
	public function __construct($siteId, $authorization)
	{
		parent::__construct('usertoken', $siteId, $authorization);
	}

	/**
	 * Get a staff user token.
	 */
	public function issue($request)
	{
		$url = $this->getFullPath('issue');

		return $this->client->post($url, $request, IssueResponse::class);
	}

	/**
	 * Revokes the user token in the Authorization header.
	 */
	public function revoke()
	{
		$url = $this->getFullPath('revoke');

		return $this->client->delete($url, null, null);
	}

}
